src/tokens/mod.rs

```diff
@@ -29,7 +29,7 @@ impl Token {
         self.children.get(0)
     }
 
-    /// Get an iterator over all children of token
+    /// Get an iterator over all children of tokens, recursively
```

Comment:
`token` was singular because it was referring to the object that is being called upon
so *of  tokens* as its used now should be after the word iterator

```rust
/// Recursively get an iterator, of tokens, over all children of `self`
```